packit/upstream.py

```diff
@@ -398,7 +404,9 @@ def _get_archive_path_from_output(self, outputs: List[str]) -> Optional[Path]:
                         self._local_project.working_dir, archive_name.strip()
                     )
                     if archive_path.is_file():
-                        logger.info(f"Created archive: {archive_path}")
+                        archive_path_absolute = archive_path.absolute()
```

Comment:
Can something go wrong if we use the absolute one now?

Reply:
linking expects relative... I had it handled by returning absolute one and then **explicitly** making it relative to the `specdir`.

Pushed just logging to see where it breaks.